package.json

```
@@ -8,15 +8,18 @@
     "brighterscript-formatter": "1.7.18"
   },
   "devDependencies": {
+    "@hulu/roca": "0.27.0",
     "@rokucommunity/bslint": "0.8.35",
     "brighterscript": "0.70.1",
+    "brs": "0.45.0",
```

```suggestion
    "@rokucommunity/brs": "0.47.4",
```

brs was forked by roku community because it's no longer maintained https://github.com/rokucommunity/brs

---

Ah, thanks! Fixed in [6efe952](https://github.com/jellyfin/jellyfin-roku/pull/543/commits/6efe952de286f911b20094ca93a9d9f9ce681e40)